assets/Treasure Hunters/README.md

```diff
@@ -0,0 +1,5 @@
+Assets in this folder are from the Treasure Hunters pack by PixelFrog
+----
+https://pixelfrog-assets.itch.io/treasure-hunters
+----
+Licence is CC0 (https://creativecommons.org/publicdomain/zero/1.0/)
```

Comment:
Sorry to be pedantic, but today we have switched from adding a README for attribution to adding a license file. See https://github.com/endlessm/threadbare/pull/88 for how to edit the `REUSE.toml` file.

Reply:
Ohh, don't worry!, and thanks!, I didn't catch that 👌🏾